### ISSUE TITLE
Review 'community' documentation - particularly Contributing.md

### ISSUE THREAD

**Issue by stevenhorsman** [confidential-containers/confidential-containers#143]
When the community repo was created in the confidential containers org, nearly two years ago (wow!), we started by copying much of the documentation from kata containers as the starter basis, which made sense, however it's clear that most of the projects are not following all the guidelines e.g. [commit format](https://github.com/confidential-containers/confidential-containers/blob/main/CONTRIBUTING.md#patch-format), so do we want to update the community documentation to: 
- Update the docs to be something that we are happy with (kata-containers felt like a good start, but it's onerous, then we should be free to improve it)
- Remove the documentation altogether and let each project come up with their own standards
- Try and start enforcing this with automation across the repos
- Something else?

Similarly some of the docs around [Continuous Integration](https://github.com/confidential-containers/confidential-containers/blob/main/CONTRIBUTING.md#continuous-integration) aren't accurate/relevant for many of the repos

This is probably a candidate for the @confidential-containers/tsc to discuss in a future meeting

**Comment by bpradipt** [comment]
> When the community repo was created in the confidential containers org, nearly two years ago (wow!), we started by copying much of the documentation from kata containers as the starter basis, which made sense, however it's clear that most of the projects are not following all the guidelines e.g. [commit format](https://github.com/confidential-containers/confidential-containers/blob/main/CONTRIBUTING.md#patch-format), so do we want to update the community documentation to:
> 
> * Update the docs to be something that we are happy with (kata-containers felt like a good start, but it's onerous, then we should be free to improve it)
> * Remove the documentation altogether and let each project come up with their own standards
> * Try and start enforcing this with automation across the repos
> * Something else?
> 
> Similarly some of the docs around [Continuous Integration](https://github.com/confidential-containers/confidential-containers/blob/main/CONTRIBUTING.md#continuous-integration) aren't accurate/relevant for many of the repos
> 
> This is probably a candidate for the @confidential-containers/tsc to discuss in a future meeting

And on a separate thread @katexochen suggested about documenting the process for such decisions. This specific issue, imho, should be discussed in the community meeting on Thursdays, and a decision taken. So we can start with this issue as a test case and document the process. We can always iterate.

Tagging few members who usually host the Thursday meeting for their inputs. Cc @fitzthum @sameo @ariel-adam

**Comment by fitzthum** [comment]
Yeah could be good to discuss the contributing guides for the Thursday meeting. There are some interesting questions there like which parts should be unified across subprojects. At some point I would also like to talk about creating fancier documentation (not just github md files).

**Comment by sameo** [comment]
@bpradipt @fitzthum @katexochen @stevenhorsman I propose that we start a discussion around that on next Thursday meeting.

**Comment by fitzthum** [comment]
I think the contributing page has been completely replaced.
